algoliasearch-core/pom.xml

```diff
@@ -0,0 +1,53 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <parent>
+        <artifactId>algoliasearch</artifactId>
+        <groupId>com.algolia</groupId>
+        <version>3.0.0-SNAPSHOT</version>
+    </parent>
+
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>algoliasearch-core</artifactId>
+    <version>3.0.0-SNAPSHOT</version>
+    <packaging>jar</packaging>
+
+    <properties>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <jackson.version>2.9.8</jackson.version>
+        <slf4j.version>1.7.25</slf4j.version>
+        <jsr305.version>3.0.2</jsr305.version>
```

Comment:
I am not sure properties are needed for "slf4j" and "jsr305": their values are used only once each

Reply:
I was told it's always a good practice to have all the version in properties. Should I move them?

Reply:
Your call. On my side I really do not see the point to keep them out-of-line if they are not to be overridden on the command line or in Maven, and if they are used only in some place. I have no strong opinion about it (but I do care about homogeneity :) ). Note that we can go like the current construction, and decide later on too.